routes/oauth.js

```diff
@@ -54,8 +54,9 @@ router.get("/redirect", (req, res) => {
         },
       }).then((data) => {
         const email = JSON.parse(data.body).email;
-        gEmails.add(email);
-        res.send(`Registered ${email} for breach alerts. You may now close this window/tab.`);
+        Subscribers.addUser(email).then(() => {
```

Comment:
Need to file an issue to replace then() with async/await as much as possible. This particular case is getting very nested and definitely needs to be refactored to chain the thens instead of nesting.

Reply:
I believe this was filed as part of #30; _"Resolve code style inconsistencies"_.

Reply:
And on that note, it may be nice to add some additional error checking here when we refactor in case of an error. Currently I'd say let's add a `.catch(err => {...})` to handle promise chain rejections/errors, but I guess post `async..await` we'd just use good old `try..catch` block to handle the unexpected.